hivemind/optim/decentralized_optimizers.py

```diff
@@ -49,19 +65,60 @@ def __init__(self, opt: torch.optim.Optimizer, dht: DHT, *, prefix: str, target_
                                          target_group_size=target_group_size, **kwargs)
         self.lock_parameters, self.update_event, self.stop_event = Lock(), Event(), Event()
 
+        if scheduler_cls:
+            self.scheduler = scheduler_cls(opt)
```

Comment:
request: make sure self.cheduler is initialized with None if scheduler_cls is not provided